layouts/partials/widgets/experiences.html

```diff
@@ -0,0 +1,20 @@
+{{ $experiences := $.Param "experiences"}}
+{{ range $experiences }}
+    <div class="mb-6">
+        <div class="bg-secondary-bg rounded border hover:shadow-lg transition ease-in-out duration-200 px-6 pt-6 pb-4">
+            <div class="flex flex-col-reverse lg:flex-row justify-between">
+                <div class="w-full lg:w-2/3">
```

Comment:
These two lines are not necessary I think. On the contrary, it will cause wrong styles, such as w-2/3.

Reply:
I see what you mean! I removed these two lines, and it fixed some width issues I was having. I will remove these as per your suggestion.